readme.md

```diff
@@ -0,0 +1,182 @@
+# Robotic Vision Integration Package (RVIP) Documentation
+![Build Status](https://img.shields.io/gitlab/pipeline/cardboardcode/rvip)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![codecov](https://codecov.io/gl/cardboardcode/rvip_ros/branch/master/graph/badge.svg?token=9XFJvJLbpK)](https://codecov.io/gl/cardboardcode/rvip)
+
+## What Is This?
+
+**RVIP** is a ROS package that offers **easy integration of 2D-3D hybrid pose alignment processes** used in robotic vision applications.
+
+By taking in both **2D RGB images** with **3D PointCloud** information, RVIP combines the high effieciency and accuracy of 2D computer vision architectures with the robust data points of depth information.
+
+This is done so in an effort to ensure higher reliability and reduced volatility of object localization results, needed for safer robot-human collaboration spaces.
+
+For a detailed documentation on RVIP, please execute the following commands:
+1. Clone this repository into the `src` folder of your catkin workspace.
+
+> ` $ git clone https://github.com/cardboardcode/rvip.git`
+
+2. Install **rosdoc_lite**.
+
+> `$ sudo apt install ros-melodic-rosdoc-lite`
+
+3. Go to the `rvip` subdirectory and run rosdoc_lite command
+
+> `$ cd rvip && rosdoc_lite .`
+
+4. View the documentation on your browser.
+
+> `$ firefox doc/html/doc/index.html `
```

Comment:
Suggestion: show a plain listing of the command instead (in a `bash` block). If you want, add Bash comments in between the commands (ie: `# this is a comment`).

Reply:
Updated. [See Commit.](https://github.com/cardboardcode/rvip-1/commit/506faa0230c341de9effe074b9283517de03d630)